scalableminds/webknossos
# Zarr streaming editable mapping support

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Go to a volume annotation with a mapping file in webknossos
- edit the segmentation and create a new segment
- request a bounding box that contains that segment, e.g. `http://localhost:9000/tracings/volume/zarr/e1d6f00c-84ad-494c-9134-a9deefa4ecc7/1/0.2.4.1?token=secretSampleUserToken`
- save response and check whether it contains the new segment, e.g. in python
```
import numpy as np

f = open("file.bin", "r")
a = np.fromfile(f, dtype=np.uint32)
```

### Issues:
- fixes #6225 

------
